rules/defense_evasion/disable_apparmor_attempt.yaml

```diff
@@ -6,54 +6,43 @@
 
 # MITRE ATT&CK Technique: T1562 - Impair Defenses - https://attack.mitre.org/techniques/T1562/
 
-- name: Disable AppArmor service
-  type: Exec
-  category: defense_evasion
-  severity: low
-  description: Monitors for attenot to disable the AppArmor service. Attackers may attempt to disable the AppArmor service 
-    to evade detection and avoid possible restrictions on their actions and resources.
-  condition: (
-    (payload.filename STARTS_WITH "/usr/bin/systemctl" AND payload.argv CONTAINS "disable" AND (payload.argv CONTAINS "apparmor" OR payload.argv CONTAINS "apparmor.service")) OR
-    (payload.filename ENDS_WITH "service" AND payload.argv CONTAINS "apparmor" AND payload.argv CONTAINS "stop") OR
-    (payload.filename ENDS_WITH "chkconfig" AND payload.argv CONTAINS "apparmor" AND payload.argv CONTAINS "off")
```

Comment:
rewrite with file deleted if possible, example symlink to service deleted from /etc/systemd conf dir